packages/runtime/container-runtime/src/test/gc/gcConfigs.spec.ts

```diff
@@ -264,6 +264,29 @@ describe("Garbage Collection configurations", () => {
 				"getMetadata returned different metadata than loaded from",
 			);
 		});
+		it("Metadata Roundtrip with only tombstoneGeneration", () => {
```

Comment:
Filled a gap here while scrutinizing the "generation" tests, since we may have partners using `gcSweepGeneration` for the first time (when enabling blob-only sweep)